packages/lightning-lsp-common/src/__tests__/utils.test.ts

```diff
@@ -70,6 +70,11 @@ it('getExtension()', () => {
     expect(utils.getExtension(jsDocument)).toBe('.js');
 });
 
+it('getBasename()', () => {
```

Comment:
Nit. Can we have the test name a bit more descriptive? I see you've followed the other examples here and they don't follow the best practice, unfortunately.